### DISCUSSION TITLE
Home renovations

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#984] [General]
Currently undergoing renovations, and it's quite an intense process. We're in need of various supplies, including building materials, doors, windows, and furniture. Do you have any recommendations for specific suppliers or stores that offer a wide range of these products? Any advice would be greatly appreciated!

**Comment by MirkosMangle** [comment]
We usually get everything in the nearest warehouse. I am too lazy to check all that stuff

**Comment by hernessteve** [comment]
Good luck with the project. [Screen tight customer service](https://www.pissedconsumer.com/company/screen-tight/customer-service.html) could provide valuable assistance with your windows and doors inquiries. As for other aspects, I'm not quite sure, but it's always a good idea to do some research. Checking Google reviews and other online feedback can offer insights from customers who have firsthand experience with various products and services. Taking the time to gather information from multiple sources will help you make informed decisions about your home improvements.
